extensions/wikia/QuickTools/QuickToolsHelper.class.php

```diff
@@ -5,20 +5,26 @@ class QuickToolsHelper extends ContextSource {
 	/**
 	 * Get all pages that should be rolled back for a given user
 	 *
-	 * @param  string $user A name to check against rev_user_text
+	 * @param  string $userName A name to check against rev_user_text
 	 * @param  string $time Timestamp to revert since
 	 * @return Array  An array of page titles to revert
 	 */
-	public function getRollbackTitles( $user, $time ) {
+	public function getRollbackTitles( $userName, $time ) {
 		$dbr = wfGetDB( DB_SLAVE );
+		$user = User::newFromName($userName);
 
 		$titles = [];
 		$where = [
 			'rev_page = page_id',
 			'page_latest = rev_id',
-			'rev_user_text' => $user,
 		];
 
+		if ( $user ) {
```

Comment:
Are we checking here whether `$user` is not empty or whether it's an instance of `User` class?

Reply:
Replaced with `User::idFromName` as you suggested here: https://github.com/Wikia/app/pull/13962#discussion_r144229883